assignment/assignment-tool/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -16812,5 +16818,45 @@ protected void setScoringAgentProperties(Context context, Assignment assignment,
 			}
 		}
 	}
+	
+	/**
+	 * Called to associate a new or edited Assignment to a GB item.
+	 * @param pTitle
+	 * @param pGradePoints
+	 * @param pDueTime
+	 * @param pAddToGradbook
+	 * @param pGradebookItem
+	 */
+	private void linkAssignmentToGradebookItem(String pTitle, String pGradePoints, Date pDueTime, String pAddToGradbook, String pGradebookItem) {
+		/*
+		 * IFF an empty GB item with the same name exists, drop it and create a new one linked to the assignment.
```

Comment:
Can you move this comment into the javadoc location of the method, its not needed here.